SilKit/source/core/vasio/VAsioPeer.hpp

```diff
@@ -130,9 +140,17 @@ class VAsioPeer
     std::deque<std::vector<uint8_t>> _sendingQueue;
     ConstBuffer _currentSendingBuffer;
     std::vector<uint8_t> _currentSendingBufferData;
+    std::vector<uint8_t> _aggregatedMessages;
 
     std::atomic_bool _sending{false};
     Core::ServiceDescriptor _serviceDescriptor;
+
+    bool _useAggregation{false};
+    const size_t _aggregationBufferThreshold{100 * 1000};
+
+    std::unique_ptr<ITimer> _flushTimer;
+    std::chrono::milliseconds _flushTimeout{50};
```

Comment:
```suggestion
    const std::chrono::milliseconds _flushTimeout{50};
```
that's actually a very important number. Just in case this comes up during debugging simulations runs or from user feedback :-)